core-rust/build.gradle

```diff
@@ -104,17 +104,17 @@ task buildRustDebug(type: Exec) {
 
 task buildRustForDocker(type: Exec) {
     def target = "x86_64-unknown-linux-gnu"
-    def docker_plattform = "linux/amd64"
+    def dockerPlattform = "linux/amd64"
```

Comment:
Super minor - but Plattform should be Platform

Reply:
It's german. :P